sdf/_embeds/dataflows/arrow_state.yaml

```
@@ -54,8 +54,7 @@ services:
           - operator: map 
             run: |
               fn new_input(_input: String) -> Result<String> {
-                let track = tracker();
-                let trackrow = track.sql(&format!("select * from `tracker`"))?;
+                let trackrow = sql(&format!("select * from `tracker`"))?;
```

probably don't need the ``
```suggestion
                let trackrow = sql(&format!("select * from tracker"))?;
```